homework/3_first_interactive_component.md

```diff
@@ -16,4 +16,4 @@ For main project (current index.html):
 
 ### Submission:
 Don't forget to push to github!
-If you want to refresh knowlage about git workflow please follow these slides: <slides link here>
\ No newline at end of file
+If you want to refresh knowlage about git workflow please follow these slides: <slides link here>
```

Comment:
link shall be added: https://mrtim.github.io/js-munich-2020-spring/lessons/0_workflow/homework_workflow

Reply:
or https://mrtim.github.io/js-munich-2020-spring/lessons/2_git_time/

Reply:
updated